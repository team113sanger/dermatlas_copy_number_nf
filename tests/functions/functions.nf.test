
nextflow_function {

    name "Test Functions"
    script "functions/functions.nf"

	

    test("Test Function filter_ascat_qc_fails") {

        function "filter_ascat_qc_fails"

        when {
            function {
                """
                // define inputs of the function here. Example:
                input[0] = ['Tbc': 'tbc']
                input[1] = file("$baseDir/tests/testdata/ascat_low_qual.list")
                """
            }
        }

        then {
            assert function.success
            assert snapshot(function.result).match()
        }

    }

	

}