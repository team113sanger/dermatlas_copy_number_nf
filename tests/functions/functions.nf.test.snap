{
    "Test Function filter_ascat_qc_fails": {
        "content": null,
        "meta": {
            "nf-test": "0.8.4",
            "nextflow": "23.10.1"
        },
        "timestamp": "2024-05-21T14:15:40.991033"
    }
}